src/main/java/bio/terra/service/load/LoadDao.java

```diff
@@ -72,83 +78,94 @@ public LoadDao(NamedParameterJdbcTemplate jdbcTemplate) {
    *
    * @param loadTag tag identifying this load
    * @param flightId flight id taking the lock
+   * @param datasetId dataset to which this load is being applied
```

Comment:
I think the javadoc can be further updated to describe how `datasetId` is used

Reply:
Good shout -- updated.